### PR TITLE
fix: Update signinText and signuptext to logo Url in oidc config

### LINKED ISSUE #1358: Update signinText and signuptext to logos
logo + text thing.

changes required:
- hub-auth update to change the provider config json
- update wallet ui to display images and remove text
- update wallet tests
- update sandbox configuration

[wallet-signup](https://user-images.githubusercontent.com/7862595/146793520-3971eff2-6438-4f7d-aaf1-1f5b0e949967.png)

wallet-signup

### PR DESCRIPTION
**Pull request by talwinder50**
Screen Shot 2021-12-21 at 1 13 09 PM

Screen Shot 2021-12-21 at 1 13 17 PM

closes #1358

Images are svg optimized

Signed-off-by: talwinder50 <talwinderkaur50@gmail.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: test/fixtures/wallet-web/.env [hunk 1]
```diff
@@ -61,8 +61,8 @@ EDV_DATABASE_URL=mongodb://mongodb.example.com:27017
 EDV_DATABASE_PREFIX=edv_
 
 # HUB-AUTH
-AUTH_REST_IMAGE=ghcr.io/trustbloc/hub-auth
-AUTH_REST_IMAGE_TAG=0.1.7
```
Comment (birtony) [inline test/fixtures/wallet-web/.env:65]: Is this supposed to be `trustbloc-cicd`? 
cc @sudeshrshetty

File: test/fixtures/wallet-web/oidc-config/demo_providers.yaml [hunk 2]
```diff
@@ -10,6 +10,6 @@ providers:
     clientID: client-id
     clientSecret: client-secret
     name: Demo OIDC
-    signUpText: Demo Sign-Up Partner
-    signInText: Demo Sign-In Partner
-    logoURL: https://hub-auth.trustbloc.local:8044/static/images/demo_provider.svg
+    signInLogoURL: https://hub-auth.trustbloc.local:8044/static/images/signin_logo.svg
+    signUpLogoURL: https://hub-auth.trustbloc.local:8044/static/images/signup_logo.svg
+    order: 1
```
Comment (birtony) [inline test/fixtures/wallet-web/oidc-config/demo_providers.yaml:15]: Missing trailing newline

File: test/fixtures/wallet-web/.env (hunk 1 above)
Comment (sudeshrshetty) [reply test/fixtures/wallet-web/.env:65]: yes, snapshots will be from cicd. release versions are from `trustbloc`

Comment (rolsonquadras) [reply test/fixtures/wallet-web/.env:65]: yes - for snapshots we use `trustbloc-cicd`

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**